examples/experiments/decay/research_decay.ipynb

```diff
@@ -0,0 +1,508 @@
+{
```

Comment:
<p><em>Suggested change:</em></p><p> </p><p>Root and branch pipelines for models training. First, <code>train_root</code> does data processing common for all branches and then each <code>train_pipeline</code> saves loss and learning rate values for further comparison.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/488/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>